include/sounds.h

```diff
@@ -9,9 +9,9 @@
 // Fourth Byte (Upper Nibble): More bitflags
 // Fourth Byte (Lower Nibble): Sound Status (this is set to SOUND_STATUS_PLAYING when passed to the audio driver.)
 #define SOUND_ARG_LOAD(bank, soundID, priority, flags) (\
-    ((u32) (bank) << 28) | \
-    ((u32) (soundID) << 16) | \
-    ((u32) (priority) << 8) | \
+    ((u32) (bank) << SOUNDARGS_SHIFT_BANK) | \
+    ((u32) (soundID) << SOUNDARGS_SHIFT_SOUNDID) | \
+    ((u32) (priority) << SOUNDARGS_SHIFT_PRIORITY) | \
     (flags) | \
     SOUND_STATUS_WAITING)
```

Comment:
isnt this part unnecessary now that you hardcode the OR

Reply:
Not sure what you mean, this is just removing redundancy with the hardcoded values defined 10 lines down.

Reply:
all the sounds are `| SOUND_STATUS_WAITING` now so no need to have it in the macro anymore

Reply:
The sound IDs didn't change; rather the comments themselves were wrong. I'd rather keep the flag as part of the macro because it's used 100% of the time and just makes the definitions harder to read. It should be fine as-is now that they're actually correct.

Reply:
i cant read :(